pkg/kubelet/cm/helpers_linux.go

```diff
@@ -44,3 +47,46 @@ func GetCgroupSubsystems() (*CgroupSubsystems, error) {
 		MountPoints: mountPoints,
 	}, nil
 }
+
+// GetPodResourceRequest returns the pod requests for the supported resources.
+// Pod request is the summation of resource requests of all containers in the pod.
+func GetPodResourceRequests(pod *api.Pod) api.ResourceList {
```

Comment:
nit: Why not re-use https://github.com/kubernetes/kubernetes/blob/master/pkg/api/resource_helpers.go#L165 or better refactor that and use it here?